sphinx_gallery/tests/test_full.py

```diff
@@ -377,6 +378,9 @@ def test_logging_std_nested(sphinx_app):
 
 def _assert_mtimes(list_orig, list_new, different=(), ignore=()):
     """Assert that the correct set of files were changed based on mtime."""
+    import numpy as np
+    from numpy.testing import assert_allclose
```

Comment:
Not sure if this is best solution but putting the imports in `setup` didn't work